docs/plugins/dffml_model.rst

```diff
@@ -34,59 +34,36 @@ tfdnnc
 
 Implemented using Tensorflow's DNNClassifier.
 
-.. code-block:: console
+First we create the training and testing datasets
+
+.. literalinclude:: /../model/tensorflow/examples/tfdnnc/train_data.sh
```

Comment:
If we include the whole file we'll end up taking up a giant space. And if we cut on the lines here as you've done then we end up leaving off a large portion of the data and the`EOF`, making copy pasting futile (although I like that you thought about the space it will take up on the page and decided to only take a chunk by using `:lines:`, good thinking).

We should put the wget command in the `train_data.sh` and `test_data.sh` files. We'll also have to put the `sed` command in both files then.